data/Pradumnasaraf.json

```diff
@@ -7,7 +7,7 @@
   "socials": [
     { "icon": "FaTwitter", "url": "https://twitter.com/pradumna_saraf" },
     { "icon": "FaGithub", "url": "https://github.com/Pradumnasaraf" },
-    { "icon": "website", "url": "https://pradumnasaraf.dev" }
+    { "icon": "FaLink", "url": "https://pradumnasaraf.dev" }
```

Comment:
I don't think you should update others' profiles 🤔 unless Pradumna accepts. Although it'd be better to tell the author (in this case, Pradumna) about it.

Reply:
I checked you haven't synced your branches with the latest changes which is why it's reflecting old changed. Sync then you will good to go.

<img width="857" alt="Screenshot 2023-01-10 at 10 07 00 PM" src="https://user-images.githubusercontent.com/51878265/211609903-6fd8ac46-b86a-4483-9197-0d96940a5265.png">

Reply:
> I checked you haven't synced your branches with the latest changes, which is why it's reflecting old changes. Sync then you will be good to go.
> 
> <img alt="Screenshot 2023-01-10 at 10 07 00 PM" width="857" src="https://user-images.githubusercontent.com/51878265/211609903-6fd8ac46-b86a-4483-9197-0d96940a5265.png">

I have synced. Also, please review the events I have added. Once they're added o my profile, I'd delete my other branch...I'll keep me updated with the latest updates on brain branch.